src/PersonalInfo.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+class PersonalInfo extends React.Component {
+
+  constructor() {
+    super();
+    this.state = { name:'', bgColor:''}
```

Comment:
```suggestion
    this.state = { name:'', backgroundColor:''}
```

Its clean code practice to use full name, `backgroundColor` is cleaner than `bgColor`